### DISCUSSION TITLE
Any plan to upgrade to 8.8.xx?

### DISCUSSION THREAD

**Discussion by Guanran928** [SpotCompiled/SpotilifeC#12] [Q&A] [answered]
I've heard spotilife breaks while working with Spotify 8.7+, but it seems like some people claimed with specific accounts spotilife still works.... So I am wondering will you try to compile the newest Spotify (and maybe try to fix spotilife..?)

**Comment by yodaluca23** [comment, marked as answer]
> I've heard spotilife breaks while working with Spotify 8.7+, but it seems like some people claimed with specific accounts spotilife still works.... So I am wondering will you try to compile the newest Spotify (and maybe try to fix spotilife..?)

I will not be making an update, if it requires extensive workarounds, and Spotilife is not opensource, so I can't edit the code...
